packages/graphql-yoga/src/server.ts

```diff
@@ -275,11 +277,16 @@ export class YogaServer<
             }
         : logger
 
-    const maskedErrorsOpts: UseMaskedErrorsOpts | null =
+    this.maskedErrorsOpts =
       options?.maskedErrors === false
         ? null
         : {
-            formatError,
+            formatError: yogaDefaultFormatError,
+            errorMessage: 'Unexpected error.',
+            isDev:
+              typeof process === 'object'
+                ? process.env.NODE_ENV === 'development'
+                : false,
```

Comment:
```suggestion
              isDev: globalThis?.process.env.NODE_ENV === 'development',
```

Just a nit, I would prefer this one